pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.6.7</version>
+        <version>2.7.0</version>
```

Comment:
```suggestion
        <version>2.7.3</version>
```

Reply:
Fixed in https://github.com/Informasjonsforvaltning/dataservice-publisher-sparql-service/pull/23/commits/beeafd2fef761ee904af8899e2e4510b5d26a766